.github/workflows/build.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: ["2.7.4", "3.0.6", "3.1.4", "3.2.2", "3.3.0-preview3"]
+        ruby-version: ["3.2.2", "3.3.9", "3.5.0-preview1"]
```

Comment:
Might be good for us to add a `3.4.x` in this matrix too

Reply:
added!